docs/content/posts/2023-12-20-release-1.18.1.md

```diff
@@ -0,0 +1,142 @@
+---
+title:  "Apache Flink 1.18.1 Release Announcement"
+date: "2023-12-20T00:00:00.000Z"
+aliases:
+- /news/2023/12/20/release-1.18.1.html
+  authors:
+- jingge:
+  name: "Jing Ge"
+  twitter: "jingengineer"
+---
+
+The Apache Flink Community is pleased to announce the first bug fix release of the Flink 1.18 series.
+
+This release includes 47 bug fixes, vulnerability fixes, and minor improvements for Flink 1.18.
+Below you will find a list of all bugfixes and improvements (excluding improvements to the build infrastructure and build stability). For a complete list of all changes see:
+[JIRA](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12353640).
+
+We highly recommend all users upgrade to Flink 1.18.1.
+
+# Release Artifacts
+
+## Maven Dependencies
+
+```xml
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-java</artifactId>
+  <version>1.18.1</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-streaming-java</artifactId>
+  <version>1.18.1</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-clients</artifactId>
+  <version>1.18.1</version>
+</dependency>
+```
+
+## Binaries
+
+You can find the binaries on the updated [Downloads page]({{< relref "downloads" >}}).
+
+## Docker Images
+
+* [library/flink](https://hub.docker.com/_/flink/tags?page=1&name=1.18.1) (official images)
+* [apache/flink](https://hub.docker.com/r/apache/flink/tags?page=1&name=1.18.1) (ASF repository)
+
+## PyPi
+
+* [apache-flink==1.18.1](https://pypi.org/project/apache-flink/1.18.1/)
+
+# Release Notes
+
+        Release Notes - Flink - Version 1.18.1
+
+<h2>        Bug
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-18356'>FLINK-18356</a>] -         flink-table-planner Exit code 137 returned from process
```

Comment:
Though this ticket is critical. As this is for internal test CI framework, shall we need to mentation this in the release note?

Reply:
I think we don't have to mention FLINK-18356 as it is a infra(jdk) issue instead of a flink bug according to the PR https://github.com/apache/flink/pull/23717